package.json

```diff
@@ -26,6 +26,7 @@
     "gulp-filter": "3.0.1",
     "gulp-rename": "1.2.2",
     "gulp-replace": "0.5.4",
+    "gulp-sass": "^2.3.2",
```

Comment:
Don't include `^` because it's not a precise version and in future even 2.3.3 **may** cause issues. I don't trust these things.